app/templates/_proposition_header.html

```diff
@@ -4,12 +4,12 @@
     <nav id="proposition-menu">
       <a href="/" id="proposition-name">Digital Marketplace</a>
       <ul id="proposition-links">
-        {% if current_user.email_address %}
-        <li><a class="home" href="/logout">Log out</a></li>
+        {% if current_user.is_authenticated() %}
+          <li><a href="{{ url_for('main.dashboard') }}">View your account</a></li>
+          <li><a class="home" href="/logout">Log out</a></li>
```

Comment:
It's not obvious that anything is targeting it in [the supplier app](https://github.com/alphagov/digitalmarketplace-supplier-frontend/search?q=.home&type=Code&utf8=%E2%9C%93), the [dm frontend toolkit](https://github.com/alphagov/digitalmarketplace-frontend-toolkit/search?utf8=%E2%9C%93&q=.home), or the [govuk frontend toolkit](https://github.com/alphagov/govuk_frontend_toolkit/search?utf8=%E2%9C%93&q=.home&type=Code).